src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  ContentEntry,
+  ContentRecord,
+  CubeName,
+  DashboardState,
+  DataModel,
+  deserializeDashboardState,
+  MdxSelect,
+  serializeDashboardState,
+  WidgetWithQueryState,
+} from "@activeviam/activeui-sdk-5.0";
+import { produce } from "immer";
+import { migrateCalculatedMeasuresInMdx } from "./migrateCalculatedMeasuresInMdx";
+
+interface DashboardsFolder {
+  entry: ContentEntry;
+  children: {
+    thumbnails: ContentRecord;
+    content: ContentRecord;
+    structure: ContentRecord;
+  };
+}
```

Comment:
We shouldn't need to redefine this interface: the dashboards folder is a `ContentRecord` like all other files/folders on the content server.